src/components/artwork_filter/for_sale_checkbox.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react"
+
+import Checkbox from "../checkbox"
+
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import { primary } from "../../assets/fonts"
+
+interface ForSaleCheckboxProps extends React.HTMLProps<ForSaleCheckbox> {
+  onClick?: any
+}
+
+interface ForSaleState {
+  isChecked: boolean,
+}
```

Comment:
FYI As long as they are not exported, you can just call these interfaces `Props` and `State`.